ui/config/webpack.config.js

```diff
@@ -29,6 +29,6 @@ module.exports = {
       BUCKET_PATH: env.BUCKET_PATH || '/',
     }),
     new webpack.NamedModulesPlugin(),
```

Comment:
- Add the feature to README.md
- Add the new ARG to Dockerfile.  (like BUCKET_PATH)